docs/v3/source/includes/upgrade_guide/changed_resources/_info_in_v3.md

```diff
@@ -0,0 +1,5 @@
+### Info in V3
+
+In V2, `info` provided links to external APIs, such as the authorization endpoint
+
+In V3, use `/` to get external APIs as these links have been removed from `v3/info`
```

Comment:
I wonder if it's worth explicitly describing what has stayed in v3/info and what has been moved to v3/

Is everything in v2/ info accessible with a combination of v3/ and v3/info?

The general flow of these guides seems to be:
What happened in v2
What happens in v3
What can i do to get the same info as v2?